svc-aggregation/agmessagebus/publish.go

```diff
@@ -35,12 +36,15 @@ func Publish(systemID, eventType, collectionType string) {
 	}
 
 	var event = common.Event{
-		EventID:   uuid.NewV4().String(),
-		MessageID: "ResourceEvent.1.0.3." + eventType,
-		EventType: eventType,
+		EventID:        uuid.NewV4().String(),
+		MessageID:      "ResourceEvent.1.0.3." + eventType,
+		EventTimestamp: time.Now().Format(time.RFC3339),
+		EventType:      eventType,
+		Message:        "Resource event",
```

Comment:
Update event message in accordance to what the event is about in aggregation service

Reply:
updated